packages/env/lib/config.js

```diff
@@ -206,21 +193,60 @@ module.exports = {
 				} ),
 				{ workDirectoryPath }
 			),
-			pluginSources: config.plugins.map( ( sourceString ) =>
-				parseSourceString( sourceString, {
-					workDirectoryPath,
-				} )
-			),
-			themeSources: config.themes.map( ( sourceString ) =>
-				parseSourceString( sourceString, {
-					workDirectoryPath,
-				} )
+			pluginSources: getSources( config.plugins, workDirectoryPath ),
+			muPluginsSources: getSources(
+				config[ 'mu-plugins' ],
+				workDirectoryPath
 			),
+			themeSources: getSources( config.themes, workDirectoryPath ),
```

Comment:
I'm not huge on the extra little layer of indirection that `getSources` adds. I'd be tempted to pull in lodash so that we can use [`castArray`](https://lodash.com/docs/4.17.15#castArray). I appreciate that this is straying into individual preference territory, though 😀 What do you think?

```js
pluginSources: castArray( config.plugins ).map( ( sourceString ) =>
	parseSourceString( sourceString, {
		workDirectoryPath,
	} )
),
themeSources: castArray( config.themes ).map( ( sourceString ) =>
	parseSourceString( sourceString, {
		workDirectoryPath,
	} )
),
muPluginSources: castArray( config[ 'mu-plugins' ] ).map( ( sourceString ) =>
	parseSourceString( sourceString, {
		workDirectoryPath,
	} )
),
```

Reply:
This won't work with the current implementation because we let `muPluginSources` be `string` or `string[]`. In the `string` case, the corresponding directory is mounted as `wp-content/themes` (for example). In the array case, all of the directories in the array are mounted as `wp-content/themes/dir`. So we need to be able to switch between those two cases

Reply:
Right you are!

Reply:
Right you are!